<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pagina_login</class>
 <widget class="QMainWindow" name="pagina_login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>271</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 35pt &quot;Lucida Grande&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Inicio de Sesión</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo_app">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>155</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Lucida Grande&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Correo electrónico:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Lucida Grande&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Contraseña:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="correo">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>250</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;  /* Bordes redondeados */</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>250</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;  /* Bordes redondeados */</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_login">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;  /* Bordes redondeados */
border: 2px solid;  /* Borde del botón */
padding: 10px 20px;  /* Espaciado interno */
font-size: 16px;  /* Tamaño de la fuente */
</string>
    </property>
    <property name="text">
     <string>Iniciar Sesión</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>440</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Lucida Grande&quot;;
color: rgb(235, 247, 78);
</string>
    </property>
    <property name="text">
     <string>¿Aún no cuentas con una cuenta?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_registro">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>470</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;  /* Bordes redondeados */
border: 2px solid;  /* Borde del botón */
padding: 10px 20px;  /* Espaciado interno */
font-size: 16px;  /* Tamaño de la fuente */
</string>
    </property>
    <property name="text">
     <string>Registrate Ahora</string>
    </property>
   </widget>
   <widget class="QLabel" name="cargando">
    <property name="geometry">
     <rect>
      <x>277</x>
      <y>330</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 8);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
